lib/diagnostic.md

```diff
@@ -12,7 +15,7 @@ Using the diagram in the `README` file (also in the `assets` folder), what are t
 you could return ALL list items (`li`) using jQuery? This is intentionally vague. (Assume starting at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$ol.children()
 <!-- your answer ends here -->
```

Comment:
TWO WAYS, please read carefully next time but the one is have is right.  Another answer could be $('li')